src/components/molecules/MarketStats.tsx

```diff
@@ -59,23 +82,11 @@ export default function MarketStats(): ReactElement {
 
   return (
     <div className={styles.stats} ref={ref}>
-      Total of <strong>{stats?.datasets.total}</strong> data sets & unique
-      datatokens published by <strong>{stats?.owners}</strong> accounts.
-      <br />
-      <PriceUnit
-        price={`${stats?.ocean}`}
-        small
-        className={styles.total}
-        conversion
-      />{' '}
-      and{' '}
-      <PriceUnit
-        price={`${stats?.datatoken}`}
-        symbol="datatokens"
-        small
-        className={styles.total}
-      />{' '}
-      in <strong>{stats?.datasets.pools}</strong> data set pools.
+      <Conversion price={`${totalLockedValue}`} hideApproximationSign />{' '}
+      <abbr title="Total Value Locked">TVL</abbr> (total value locked) across{' '}
+      <strong>{stats?.datasets.pools}</strong> data set pools that contain{' '}
```

Comment:
we should get the pools number also from a subgraph query, then we can completely remove fetching from `market-stats.oceanprotocol.com`

And it's fine removing `(total value locked)` from the text display since it's behind the `abbr`